signal_ffw: 0 #0 torque, 1 velocity, 2 position

multisine:
  test_namespace:
  - "test37_jnt1"
  - "test38_jnt2"
  - "test39_jnt1"
  - "test40_jnt2"
  - "test41_jnt1"
  - "test42_jnt2"
test37_jnt1:
  test_duration: 300.000000
  start_stop_transient: 0.1
  natural_frequency: [6.28318531e-02, 1.25663706e-01, 2.51327412e-01, 4.39822972e-01, 5.65486678e-01, 6.28318531e-01, 8.16814090e-01, 1.13097336e+00, 1.19380521e+00, 1.31946891e+00, 1.44513262e+00, 1.57079633e+00, 1.63362818e+00, 1.94778745e+00, 2.19911486e+00, 2.26194671e+00, 2.38761042e+00, 2.51327412e+00, 2.89026524e+00, 3.14159265e+00 ]

  sinusoidal_coefficients: 
  - [1.01773143e+01, 0.00000000e+00 ]
  - [3.57414787e+00, 0.00000000e+00 ]
  - [1.98426587e+00, 0.00000000e+00 ]
  - [-7.62456109e-01, 0.00000000e+00 ]
  - [2.83763833e+00, 0.00000000e+00 ]
  - [-1.05326523e+00, 0.00000000e+00 ]
  - [3.40984769e+00, 0.00000000e+00 ]
  - [1.45542453e+00, 0.00000000e+00 ]
  - [3.45183535e+00, 0.00000000e+00 ]
  - [-3.91934645e+00, 0.00000000e+00 ]
  - [-2.20887983e+00, 0.00000000e+00 ]
  - [3.60537829e+00, 0.00000000e+00 ]
  - [3.90689781e+00, 0.00000000e+00 ]
  - [-2.27383124e+00, 0.00000000e+00 ]
  - [3.69188340e+00, 0.00000000e+00 ]
  - [-2.24678738e+00, 0.00000000e+00 ]
  - [-2.40442707e+00, 0.00000000e+00 ]
  - [-6.63707669e-01, 0.00000000e+00 ]
  - [-3.88763398e+00, 0.00000000e+00 ]
  - [1.95253272e+00, 0.00000000e+00 ]

  cosinusoidal_coefficients: 
  - [6.02230409e+00, 0.00000000e+00 ]
  - [1.66249732e+00, 0.00000000e+00 ]
  - [3.40604159e+00, 0.00000000e+00 ]
  - [-3.86743986e+00, 0.00000000e+00 ]
  - [2.73609925e+00, 0.00000000e+00 ]
  - [-3.79856061e+00, 0.00000000e+00 ]
  - [-1.97771815e+00, 0.00000000e+00 ]
  - [3.66335499e+00, 0.00000000e+00 ]
  - [1.90348708e+00, 0.00000000e+00 ]
  - [4.20896373e-01, 0.00000000e+00 ]
  - [3.26485532e+00, 0.00000000e+00 ]
  - [-1.59363663e+00, 0.00000000e+00 ]
  - [5.24003684e-01, 0.00000000e+00 ]
  - [3.21995681e+00, 0.00000000e+00 ]
  - [-1.38145841e+00, 0.00000000e+00 ]
  - [-3.23888512e+00, 0.00000000e+00 ]
  - [3.12364544e+00, 0.00000000e+00 ]
  - [-3.88560452e+00, 0.00000000e+00 ]
  - [6.51714946e-01, 0.00000000e+00 ]
  - [3.42433151e+00, 0.00000000e+00 ]

test38_jnt2:
  test_duration: 300.000000
  start_stop_transient: 0.1
  natural_frequency: [6.28318531e-02, 1.88495559e-01, 5.02654825e-01, 5.65486678e-01, 6.28318531e-01, 6.91150384e-01, 7.53982237e-01, 1.19380521e+00, 1.63362818e+00, 1.75929189e+00, 2.01061930e+00, 2.26194671e+00, 2.38761042e+00, 2.45044227e+00, 2.57610598e+00, 2.70176968e+00, 2.82743339e+00, 2.89026524e+00, 2.95309709e+00, 3.01592895e+00 ]

  sinusoidal_coefficients: 
  - [0.00000000e+00, 1.00459295e+01 ]
  - [0.00000000e+00, 4.74461262e-01 ]
  - [0.00000000e+00, 3.31166567e+00 ]
  - [0.00000000e+00, -1.11382928e+00 ]
  - [0.00000000e+00, -1.51818539e+00 ]
  - [0.00000000e+00, 3.86886279e+00 ]
  - [0.00000000e+00, -4.03112664e+00 ]
  - [0.00000000e+00, -3.85798797e+00 ]
  - [0.00000000e+00, 1.19381994e+00 ]
  - [0.00000000e+00, -2.91585669e+00 ]
  - [0.00000000e+00, -3.14023876e+00 ]
  - [0.00000000e+00, -2.94931779e+00 ]
  - [0.00000000e+00, -3.93051888e+00 ]
  - [0.00000000e+00, -7.14433321e-01 ]
  - [0.00000000e+00, 2.81424827e+00 ]
  - [0.00000000e+00, -2.63753850e+00 ]
  - [0.00000000e+00, 4.30019640e-01 ]
  - [0.00000000e+00, 2.23833785e+00 ]
  - [0.00000000e+00, 4.00701932e+00 ]
  - [0.00000000e+00, -9.82556740e-01 ]

  cosinusoidal_coefficients: 
  - [0.00000000e+00, 7.55679491e+00 ]
  - [0.00000000e+00, 4.16332970e+00 ]
  - [0.00000000e+00, -2.56735236e+00 ]
  - [0.00000000e+00, 4.03953116e+00 ]
  - [0.00000000e+00, -3.90557816e+00 ]
  - [0.00000000e+00, -1.60944970e+00 ]
  - [0.00000000e+00, -1.14387310e+00 ]
  - [0.00000000e+00, -1.63534597e+00 ]
  - [0.00000000e+00, -4.01661818e+00 ]
  - [0.00000000e+00, -3.00935332e+00 ]
  - [0.00000000e+00, -2.77440592e+00 ]
  - [0.00000000e+00, -2.97656719e+00 ]
  - [0.00000000e+00, 1.45239422e+00 ]
  - [0.00000000e+00, -4.12892391e+00 ]
  - [0.00000000e+00, -3.10458279e+00 ]
  - [0.00000000e+00, -3.25604335e+00 ]
  - [0.00000000e+00, 4.16815436e+00 ]
  - [0.00000000e+00, 3.54235393e+00 ]
  - [0.00000000e+00, -1.22565241e+00 ]
  - [0.00000000e+00, -4.07345184e+00 ]

test39_jnt1:
  test_duration: 300.000000
  start_stop_transient: 0.1
  natural_frequency: [6.28318531e-02, 1.69646003e+00, 1.75929189e+00, 1.82212374e+00, 2.01061930e+00, 2.07345115e+00, 2.26194671e+00, 2.32477856e+00, 2.76460154e+00, 3.07876080e+00, 3.51858377e+00, 3.58141563e+00, 4.02123860e+00, 4.08407045e+00, 4.14690230e+00, 4.27256601e+00, 4.46106157e+00, 4.52389342e+00, 4.58672527e+00, 4.71238898e+00 ]

  sinusoidal_coefficients: 
  - [1.63252351e+00, 0.00000000e+00 ]
  - [-3.68903914e+00, 0.00000000e+00 ]
  - [-1.72108153e+00, 0.00000000e+00 ]
  - [-5.53576134e-01, 0.00000000e+00 ]
  - [-2.36454561e+00, 0.00000000e+00 ]
  - [3.08631695e+00, 0.00000000e+00 ]
  - [-1.29371166e+00, 0.00000000e+00 ]
  - [1.29784640e+00, 0.00000000e+00 ]
  - [3.46178143e+00, 0.00000000e+00 ]
  - [3.40559947e+00, 0.00000000e+00 ]
  - [-1.94278434e+00, 0.00000000e+00 ]
  - [2.53565926e+00, 0.00000000e+00 ]
  - [1.35812183e-01, 0.00000000e+00 ]
  - [3.77078614e+00, 0.00000000e+00 ]
  - [3.14789935e+00, 0.00000000e+00 ]
  - [3.84265264e+00, 0.00000000e+00 ]
  - [-2.19717198e-01, 0.00000000e+00 ]
  - [2.25713834e+00, 0.00000000e+00 ]
  - [-2.20122379e+00, 0.00000000e+00 ]
  - [9.42233569e-01, 0.00000000e+00 ]

  cosinusoidal_coefficients: 
  - [1.14197050e+01, 0.00000000e+00 ]
  - [1.08493255e+00, 0.00000000e+00 ]
  - [-3.43859954e+00, 0.00000000e+00 ]
  - [-3.80521246e+00, 0.00000000e+00 ]
  - [3.03232789e+00, 0.00000000e+00 ]
  - [-2.29362946e+00, 0.00000000e+00 ]
  - [3.62110460e+00, 0.00000000e+00 ]
  - [3.61962472e+00, 0.00000000e+00 ]
  - [1.67396468e+00, 0.00000000e+00 ]
  - [1.78549171e+00, 0.00000000e+00 ]
  - [-3.31838477e+00, 0.00000000e+00 ]
  - [-2.89076470e+00, 0.00000000e+00 ]
  - [-3.84286917e+00, 0.00000000e+00 ]
  - [7.53166854e-01, 0.00000000e+00 ]
  - [-2.20835190e+00, 0.00000000e+00 ]
  - [1.41806726e-01, 0.00000000e+00 ]
  - [-3.83898590e+00, 0.00000000e+00 ]
  - [3.11310374e+00, 0.00000000e+00 ]
  - [-3.15288792e+00, 0.00000000e+00 ]
  - [3.72804028e+00, 0.00000000e+00 ]

test40_jnt2:
  test_duration: 300.000000
  start_stop_transient: 0.1
  natural_frequency: [6.28318531e-02, 1.75929189e+00, 1.82212374e+00, 1.94778745e+00, 2.45044227e+00, 2.70176968e+00, 2.82743339e+00, 3.07876080e+00, 3.33008821e+00, 3.45575192e+00, 3.64424748e+00, 3.76991118e+00, 3.83274304e+00, 3.89557489e+00, 4.08407045e+00, 4.14690230e+00, 4.52389342e+00, 4.58672527e+00, 4.64955713e+00, 4.71238898e+00 ]

  sinusoidal_coefficients: 
  - [0.00000000e+00, -9.62328077e+00 ]
  - [0.00000000e+00, 3.71532348e+00 ]
  - [0.00000000e+00, 9.21873317e-01 ]
  - [0.00000000e+00, -2.54814672e+00 ]
  - [0.00000000e+00, 7.43582880e-01 ]
  - [0.00000000e+00, -2.89904654e+00 ]
  - [0.00000000e+00, 3.39953182e+00 ]
  - [0.00000000e+00, -2.89826072e+00 ]
  - [0.00000000e+00, 3.31303257e+00 ]
  - [0.00000000e+00, -2.84074059e+00 ]
  - [0.00000000e+00, 3.48725913e+00 ]
  - [0.00000000e+00, 3.15742155e+00 ]
  - [0.00000000e+00, 1.13008662e+00 ]
  - [0.00000000e+00, -3.53480656e+00 ]
  - [0.00000000e+00, -9.26782264e-01 ]
  - [0.00000000e+00, -3.57099839e+00 ]
  - [0.00000000e+00, 1.74837379e+00 ]
  - [0.00000000e+00, 2.02441682e-01 ]
  - [0.00000000e+00, -3.19640876e+00 ]
  - [0.00000000e+00, -1.26983992e+00 ]

  cosinusoidal_coefficients: 
  - [0.00000000e+00, -5.72748672e+00 ]
  - [0.00000000e+00, 3.61936131e-01 ]
  - [0.00000000e+00, 3.61728847e+00 ]
  - [0.00000000e+00, -2.72792497e+00 ]
  - [0.00000000e+00, -3.65810208e+00 ]
  - [0.00000000e+00, -2.35162826e+00 ]
  - [0.00000000e+00, 1.54201484e+00 ]
  - [0.00000000e+00, -2.35259667e+00 ]
  - [0.00000000e+00, -1.72001207e+00 ]
  - [0.00000000e+00, -2.42173889e+00 ]
  - [0.00000000e+00, 1.33178456e+00 ]
  - [0.00000000e+00, -1.99130999e+00 ]
  - [0.00000000e+00, -3.55774234e+00 ]
  - [0.00000000e+00, -1.19990370e+00 ]
  - [0.00000000e+00, 3.61603387e+00 ]
  - [0.00000000e+00, -1.08747266e+00 ]
  - [0.00000000e+00, -3.29815333e+00 ]
  - [0.00000000e+00, 3.72741783e+00 ]
  - [0.00000000e+00, 1.92810719e+00 ]
  - [0.00000000e+00, 3.51028958e+00 ]

test41_jnt1:
  test_duration: 300.000000
  start_stop_transient: 0.1
  natural_frequency: [6.28318531e-02, 3.58141563e+00, 3.70707933e+00, 3.76991118e+00, 3.95840674e+00, 4.08407045e+00, 4.33539786e+00, 4.46106157e+00, 4.52389342e+00, 4.58672527e+00, 4.83805269e+00, 4.90088454e+00, 4.96371639e+00, 5.27787566e+00, 5.34070751e+00, 5.46637122e+00, 5.52920307e+00, 5.59203492e+00, 5.65486678e+00, 5.96902604e+00 ]

  sinusoidal_coefficients: 
  - [-7.51246372e+00, 0.00000000e+00 ]
  - [4.05723991e-01, 0.00000000e+00 ]
  - [2.70798842e+00, 0.00000000e+00 ]
  - [3.58784213e+00, 0.00000000e+00 ]
  - [3.10227137e+00, 0.00000000e+00 ]
  - [-3.30573754e+00, 0.00000000e+00 ]
  - [3.84540196e+00, 0.00000000e+00 ]
  - [-3.76120154e+00, 0.00000000e+00 ]
  - [3.75267833e-01, 0.00000000e+00 ]
  - [-8.89270450e-02, 0.00000000e+00 ]
  - [8.37163239e-01, 0.00000000e+00 ]
  - [-3.85984667e+00, 0.00000000e+00 ]
  - [3.84944356e+00, 0.00000000e+00 ]
  - [-8.16987795e-01, 0.00000000e+00 ]
  - [3.09699947e-01, 0.00000000e+00 ]
  - [3.74670877e+00, 0.00000000e+00 ]
  - [3.78538336e+00, 0.00000000e+00 ]
  - [-3.35643987e+00, 0.00000000e+00 ]
  - [2.71287537e+00, 0.00000000e+00 ]
  - [3.72100202e+00, 0.00000000e+00 ]

  cosinusoidal_coefficients: 
  - [8.84601697e+00, 0.00000000e+00 ]
  - [-3.84718685e+00, 0.00000000e+00 ]
  - [-2.76265404e+00, 0.00000000e+00 ]
  - [-1.44666770e+00, 0.00000000e+00 ]
  - [2.31114062e+00, 0.00000000e+00 ]
  - [2.00936753e+00, 0.00000000e+00 ]
  - [-4.22305976e-01, 0.00000000e+00 ]
  - [-9.04887619e-01, 0.00000000e+00 ]
  - [-3.85027696e+00, 0.00000000e+00 ]
  - [-3.86749927e+00, 0.00000000e+00 ]
  - [-3.77685270e+00, 0.00000000e+00 ]
  - [2.58925212e-01, 0.00000000e+00 ]
  - [3.83722419e-01, 0.00000000e+00 ]
  - [3.78126824e+00, 0.00000000e+00 ]
  - [3.85610484e+00, 0.00000000e+00 ]
  - [-9.63136529e-01, 0.00000000e+00 ]
  - [-7.97703835e-01, 0.00000000e+00 ]
  - [1.92347863e+00, 0.00000000e+00 ]
  - [-2.75785530e+00, 0.00000000e+00 ]
  - [-1.05811274e+00, 0.00000000e+00 ]

test42_jnt2:
  test_duration: 300.000000
  start_stop_transient: 0.1
  natural_frequency: [6.28318531e-02, 3.26725636e+00, 3.58141563e+00, 3.64424748e+00, 3.76991118e+00, 4.14690230e+00, 4.20973416e+00, 4.27256601e+00, 4.33539786e+00, 4.58672527e+00, 4.64955713e+00, 4.71238898e+00, 4.83805269e+00, 5.27787566e+00, 5.40353936e+00, 5.52920307e+00, 5.65486678e+00, 5.78053048e+00, 6.03185789e+00, 6.22035345e+00 ]

  sinusoidal_coefficients: 
  - [0.00000000e+00, 8.65636769e+00 ]
  - [0.00000000e+00, 3.56616478e+00 ]
  - [0.00000000e+00, -2.68149141e-01 ]
  - [0.00000000e+00, 1.29947613e+00 ]
  - [0.00000000e+00, -2.34574402e+00 ]
  - [0.00000000e+00, -3.65960605e+00 ]
  - [0.00000000e+00, 3.43826432e+00 ]
  - [0.00000000e+00, 1.55796750e+00 ]
  - [0.00000000e+00, 2.91419914e+00 ]
  - [0.00000000e+00, 1.56878488e+00 ]
  - [0.00000000e+00, -9.19951661e-01 ]
  - [0.00000000e+00, -3.67004612e+00 ]
  - [0.00000000e+00, 2.78237967e+00 ]
  - [0.00000000e+00, 1.61730683e+00 ]
  - [0.00000000e+00, -1.38500769e+00 ]
  - [0.00000000e+00, -2.35727976e+00 ]
  - [0.00000000e+00, 3.28311803e+00 ]
  - [0.00000000e+00, 3.27323251e+00 ]
  - [0.00000000e+00, -2.48486333e+00 ]
  - [0.00000000e+00, -2.79137153e+00 ]

  cosinusoidal_coefficients: 
  - [0.00000000e+00, -6.80749884e+00 ]
  - [0.00000000e+00, 8.70310351e-01 ]
  - [0.00000000e+00, 3.66101999e+00 ]
  - [0.00000000e+00, 3.43312294e+00 ]
  - [0.00000000e+00, 2.82355385e+00 ]
  - [0.00000000e+00, 2.86801105e-01 ]
  - [0.00000000e+00, -1.28581096e+00 ]
  - [0.00000000e+00, -3.32380935e+00 ]
  - [0.00000000e+00, -2.23213232e+00 ]
  - [0.00000000e+00, -3.31871743e+00 ]
  - [0.00000000e+00, -3.55368264e+00 ]
  - [0.00000000e+00, 7.57152545e-02 ]
  - [0.00000000e+00, -2.39443829e+00 ]
  - [0.00000000e+00, 3.29534368e+00 ]
  - [0.00000000e+00, -3.39951836e+00 ]
  - [0.00000000e+00, 2.81393026e+00 ]
  - [0.00000000e+00, -1.64198275e+00 ]
  - [0.00000000e+00, -1.66160171e+00 ]
  - [0.00000000e+00, 2.70192997e+00 ]
  - [0.00000000e+00, 2.38394973e+00 ]

